packages/core/src/utils/AssetUtil.ts

```diff
@@ -1,7 +1,15 @@
+import type { Chain } from '@web3modal/common'
 import { ApiController } from '../controllers/ApiController.js'
 import { AssetController } from '../controllers/AssetController.js'
 import type { CaipNetwork, Connector, WcWallet } from './TypeUtil.js'
 
+const chainImageIds: Record<Chain, string> = {
```

Comment:
Make sure we are using these, it could be something not necessary right now

Reply:
Id rename to namespace see prev prev comment